File: ppg-testing-library/Identifiability/I7-identifying-retrieved-data/Python/client1.py
#!/usr/bin/env python3

import requests

def post_data():
    url = 'http://test.com/data'
    #@Identifier
    personal_data = {'name': 'firstname lastname'}
    requests.post(url, json = personal_data)

if __name__ == '__main__':
    post_data()

File: ppg-testing-library/Detectability/D5-detectable-at-retrieval/Go/client1.go
package main

import (
	"net/http"
)

func main() {
    //@Identifier
    name := "firstname lastname"
	data := url.Values{
		"Name": {name},
		"Message": {"helloworld"},
	}

	http.PostForm("http://test.com/data", data)
}

